qemu/tests/cfg/block_with_iommu.cfg

```diff
@@ -18,6 +18,7 @@
             virtio_dev_disable_modern = off
             check_key_words = "DMAR: IOMMU enabled;"
             check_key_words += "DMAR: Intel(R) Virtualization Technology for Directed I/O"
+            dd_cmd = "dd if=/dev/urandom of=/home/dd_test oflag=direct bs=1M count=10"
```

Comment:
It just  declare only, it needs to execute this in guest.

Reply:
Check L64 it's already executing it

Reply:
Could you please do it again after reboot (line71),  other part  LGTM

Reply:
Done !